src/enochecker/enochecker.py

```diff
@@ -490,7 +490,18 @@ def run(self, method: Optional[Union[str, Callable]] = None) -> CheckerResult:
                 ),
                 exc_info=eno,
             )
-            return CheckerResult.from_exception(eno)  # , eno.message
+
+            if (
+                self.flag
+                and self.flag in str(eno)
+                or (eno.message and self.flag in eno.message)
```

Comment:
This fails with `Unsupported operand types for in ("Optional[str]" and "str")`

Reply:
What fails? And why?

Reply:
See https://dev.azure.com/ENOFLAG/ENOWARS/_build/results?buildId=1843&view=results